schema_cache.rst

```diff
@@ -0,0 +1,57 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+Users are often confused by PostgREST's database schema cache. It is present because detecting foreign key relationships between tables (including how those relationships pass through views) is necessary, but costly. API requests consult the schema cache as part of :ref:`resource_embedding`. However if the schema changes while the server is running it results in a stale cache and leads to errors claiming that no relations are detected between tables.
+
+.. important::
+
+   Since v5.0, PostgREST also makes use of the schema cache for stored functions metadata: parameters, return type, volatility.
+   It also uses the schema cache for resolving overloaded functions. You should refresh the cache if a change in any of the prior is done.

```

Comment:
I think this note should be merged with the above paragraph now. It's been a while since v5.0.